packages/client/hmi-client/src/model-representation/petrinet/mira-petri.ts

```diff
@@ -119,6 +119,9 @@ export const getTransitions = (amr: Model, lookup: Map<string, string>) => {
 		} else {
 			obj.base = existingTransition.id;
 		}
+		obj.input = transition.input;
+		obj.output = transition.output;
+
```

Comment:
These are a bit dangerous because we don't know if there may be some strata called `input` or `output`, since you need an AMR instance in `createParameterMatrix` anyway ... it would be better to make the construction there instead.